.github/workflows/test.yml

```diff
@@ -71,7 +70,6 @@ jobs:
           echo "${PATH}"
           which pip
           which python
-          python -m pip install -r requirements.txt
```

Comment:
pretty sure we need this for the poetry test dep?

Reply:
I removed it since `requirements-dev.txt` contains the line

```
-r requirements.txt
```

But perhaps we should instead remove that line and explicitly install those separately.

Or even better, we may want to leverage Hatch environments. But I'd prefer to do that in a separate PR. I'm trying to limit the current scope to switching from setuptools.